git_machete/utils.py

```diff
@@ -94,7 +94,21 @@ def debug(msg: Optional[str] = None) -> None:
     if debug_mode:
         function_name = bold(inspect.stack()[1].function)
         args, _, _, values = inspect.getargvalues(inspect.stack()[1].frame)
-        function_args = bold(f"({', '.join([arg+'='+str(values[arg]) if arg != 'self' else 'self' for arg in args])})")
+
+        redact_arg_by_name = ['token']
+        for arg in redact_arg_by_name:
+            values[arg] = '***'
+
+        redact_arg_by_value = ['ghp_']
+        for arg, value in values.items():
+            if value and isinstance(value, str):
+                if any(value.startswith(arg_) for arg_ in redact_arg_by_value):
+                    values[arg] = '***'
+
+        excluded_args = {'self'}
+        allowed_args = [arg for arg in args if arg not in excluded_args]
```

Comment:
I'd say just `included_args`, as `self` isn't somehow _disallowed_, it's just _excluded_

Reply:
Although TBH... `self` is sometimes useful to include as well, did you exclude it due to security reasons? 🤔

Reply:
No, i just did to for visual reasons but also `self` doesn't provide that much information, @PawelLipski do you want me to allow it?

Reply:
I suppose that in >90% of logs `self` with be an instance of `MacheteClient` or `GitContext` which indeed brings little value... amirite? if so, then exclude `self` indeed